### PR TITLE
LINDA--Updated Gas Datums and Atmospheric Defines

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Ok...short end of it:

- Updates Gas Datums to TG's latest standards
- Uptdates Atmospherics defines to TG's latest standards
 - This means that clothing has some different define names, but the overall effects are the same.

There's still some quirks about LINDA, even with this,but this should help eleminate some of the differences and help us pin down remaining bugs.

Needs a lot of scrutiny and look-through by @Tigercat2000

No runtimes or odd behaviors that I could see, initially.

### CODE REVIEW

File: code/__DEFINES/atmospherics.dm [hunk 1]
```diff
@@ -15,123 +14,105 @@
...
+// Atmos pipe limits
+#define MAX_OUTPUT_PRESSURE					4500 // (kPa) What pressure pumps and powered equipment max out at.
+#define MAX_TRANSFER_RATE					200 // (L/s) Maximum speed powered equipment can work at.
 
-#define TANK_FRAGMENT_PRESSURE	(50.*ONE_ATMOSPHERE) // Boom 3x3 base explosion
-#define TANK_FRAGMENT_SCALE	    (10.*ONE_ATMOSPHERE) // +1 for each SCALE kPa aboe threshold
-								// was 2 atm
\ No newline at end of file
+// Sound Limits
+#define SOUND_MINIMUM_PRESSURE 10
```
Comment (tigercat2000) [inline code/__DEFINES/atmospherics.dm]: :+1: to all of this- I intended to clean it out at some point, it was just copypasted from setup.dm

File: code/datums/gas_mixture.dm [hunk 2]
```diff
@@ -61,901 +57,927 @@ What are the archived variables for?
...
-		var/delta_temperature = (temperature_archived - sharer.temperature_archived)
-		if(abs(delta_temperature) > MINIMUM_TEMPERATURE_DELTA_TO_CONSIDER)
-			var/self_heat_capacity = heat_capacity_archived()
-			var/sharer_heat_capacity = sharer.heat_capacity_archived()
+/datum/gas_mixture/temperature_turf_share(turf/simulated/sharer, conduction_coefficient)
+	var/delta_temperature = (temperature_archived - sharer.temperature)
+	if(abs(delta_temperature) > MINIMUM_TEMPERATURE_DELTA_TO_CONSIDER)
+		var/self_heat_capacity = heat_capacity()
 
-			if((sharer_heat_capacity > MINIMUM_H
```
Comment (tigercat2000) [inline code/datums/gas_mixture.dm]: This looks like it is mostly just un-relative-pathing, and I think I see something involved with fire?
Should be fine.

**Comment by tigercat2000** [comment]
@Fox-McCloud As far as I can see, the entire gas datum file is more or less the same, with some optimizations for fire/heat.

**Comment by Fox-McCloud** [comment]
Yeah, I didn't do a 100% line by line, but from what I can tell, not much has really changed from FEA gas mixture datums--just like..maybe 2-3 things?

**Comment by tigercat2000** [comment]
Yeah, it should be fine, if you didn't get any runtimes.

**Comment by Fox-McCloud** [comment]
Either case, once this is merged hopefully we'll be able to pin down the last few remaining LINDA bugs.

File: code/datums/gas_mixture.dm (hunk 2 above)
Comment (RemieRichards) [inline code/datums/gas_mixture.dm]: Also contains an equation from human breathing that I made into a generic proc (literally above your comment) and a proc I made to reverse that equation, plus proofs.

Comment (Fox-McCloud) [inline code/datums/gas_mixture.dm]: Yup. We don't use it (ATM), but I included it as a "just in case" sorta thing.

**Pull request merged**